test/Dialect/FIRRTL/lower-layers.mlir

```diff
@@ -353,24 +368,32 @@ firrtl.circuit "Test" {
   // CHECK-NEXT:     sv.ifdef @layer$Inline$Inline {
   // CHECK-NEXT:       %w2 = firrtl.wire
   // CHECK-NEXT:     }
+  // CHECK-NEXT:     firrtl.instance inline_bound {{.*}} @ModuleWithInlineLayerBlocks_Inline_Bound()
   // CHECK-NEXT:   }
+  // CHECK-NEXT:   firrtl.instance bound_inline_bound
+  // CHECK-NEXT:   firrtl.instance bound {{.*}} @ModuleWithInlineLayerBlocks_Bound()
   // CHECK-NEXT: }
   firrtl.module @ModuleWithInlineLayerBlocks() {
     firrtl.layerblock @Inline {
       %w1 = firrtl.wire : !firrtl.uint<1>
       firrtl.layerblock @Inline::@Inline {
         %w2 = firrtl.wire : !firrtl.uint<2>
       }
+      firrtl.layerblock @Inline::@Bound {
+        %w3 = firrtl.wire : !firrtl.uint<3>
+      }
     }
 
     firrtl.layerblock @Bound {
-      %w3 = firrtl.wire : !firrtl.uint<3>
+      %w4 = firrtl.wire : !firrtl.uint<4>
       firrtl.layerblock @Bound::@Inline {
-        %w4 = firrtl.wire : !firrtl.uint<4>
+        %w5 = firrtl.wire : !firrtl.uint<5>
+        firrtl.layerblock @Bound::@Inline::@Bound {
+          %w6 = firrtl.wire : !firrtl.uint<6>
+        }
       }
     }
   }
-
```

Comment:
Nit: unrelated whitespace change. Consider removing this or committing separately.

Reply:
Gone